migrations/groups/update-groups-with-group-plans.js

```diff
@@ -0,0 +1,20 @@
+import Bluebird from 'bluebird';
+
+import { model as Group } from '../../website/server/models/group';
+import * as payments from '../../website/server/libs/payments';
+
+async function updateGroupsWithGroupPlans () {
+    let groups = await Group.find({
+      'purchased.plan.customerId': 'group-unlimited',
+    }).exec();
+
+    let promises = [];
+    groups.forEach(function(group) {
+      promises.push(payments.addSubscriptionToGroupUsers(group));

```

Comment:
Will this output any errors if any of the subscriptions fail? E.g., for the reasons listed [here](https://habitica.slack.com/archives/group-plans/p1488878227000105). If so, whoever runs this can pass on errors to me for me to fix the subscriptions manually.

If it won't output errors, we can manually check that all group plan members have free subscriptions. Let me know if that will be needed.